src/main/java/org/mule/extension/aggregator/internal/privileged/executor/AbstractAggregatorExecutor.java

```diff
@@ -213,6 +222,30 @@ public void start() throws MuleException {
     }
   }
 
+  private void executeDeferActions() {
+    quorumManager = new Thread(() -> {
+      boolean startCompleted = false;
+      while (!startCompleted) {
+        try {
+          Thread.sleep(200);
```

Comment:
Yep, this should also be configurable. 
Also, I am thinking if an Sleep is the best option. For what I can remember, we were against using sleep() for waiting.
Check Javadoc of ScheduledExecutorService. There is a recommended way of scheduling a periodic task to be run for some time.

Reply:
We moved the handling of this logic to SchedulerService, thanks a lot for the suggestion.